### DISCUSSION TITLE
希望能上架微软商店，或者添加自动更新模块

### DISCUSSION THREAD

**Discussion by RicardoScofileld** [cangzhang/ChampR#158] [General]
**Describe the bug 详细描述**

**Screenshots or screen record 截图或录屏**

**Desktop (please complete the following information) 系统环境:**

**Additional context 附加信息**

**Comment by cangzhang** [comment]
会考虑考虑

**Comment by TOP-LH** [comment]
> 会考虑考虑

请务必添加更新的操作, 老是手动拉太麻烦了, 提示框点击升级那种! thank!

**Comment by JamYiz** [comment]
大佬考虑的怎么样了😂😂
